openstack/nova/values.yaml

```diff
@@ -744,6 +744,23 @@ mysql_metrics:
         UNION ALL SELECT 'i-uuid', 'source', 'dest', 0;
       values:
         - "gauge"
+    - help: Multiple Volumes Attachments
```

Comment:
Can we call them or add (BDM) so it's clear that we're not talking about volume-attachments from Cinder?

Reply:
Sure, I'll change it.